tests/test_point/test_point_arithmetic.py

```diff
@@ -0,0 +1,106 @@
+import contextlib
+
+import pytest
+from cryptography.hazmat.backends.openssl import backend
+from cryptography.hazmat.primitives.asymmetric import ec
+
+from umbral.bignum import BigNum
+from umbral.config import default_curve
+from umbral.point import Point
+
+curve = default_curve() or ec.SECP256K1()
```

Comment:
Let's just have `CURVE = default_curve()` here.

Reply:
I ended up removing it completely, and letting the default pass through.